### PR TITLE
DEVX-1542 Update ksqlDB Transforming tutorial to 0.7 syntax and expected outputs

### PR DESCRIPTION
**Pull request by rspurgeon**

### CODE REVIEW

File: _includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log [hunk 1]
```diff
@@ -1,6 +1,9 @@
-294 | Die Hard | 1988 | action
-354 | Tree of Life | 2011 | drama
-782 | A Walk in the Clouds | 1995 | romance
-128 | The Big Lebowski | 1998 | comedy
++--------------------------------------------+--------------------------------------------+--------------------------------------------+--------------------------------------------+
```
Comment (colinhicks) [inline _includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log]: Looks like the corresponding expected output needs to change in
`tutorial-steps/dev/expected-transient.log`

+------------+------------+------------+------------+
|ID          |TITLE       |YEAR        |GENRE       |
+------------+------------+------------+------------+
|294         |Die Hard    |1988        |action      |
|354         |Tree of Life|2011        |drama       |
|782         |A Walk in th|1995        |romance     |
|            |e Clouds    |            |            |
|128         |The Big Lebo|1998        |comedy      |
|            |wski        |            |            |

Comment (rspurgeon) [reply _includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log]: done, thanks for the tips on this

File: _includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-|ID                                          |TITLE                                       |YEAR                                        |GENRE                                       |
-+--------------------------------------------+--------------------------------------------+--------------------------------------------+--------------------------------------------+
-|294                                         |Die Hard                                    |1988                                        |action                                      |
-|354                                         |Tree of Life                                |2011                                        |drama                                       |
-|782                                         |A Walk in the Clouds                        |1995                                        |romance                                     |
-|128                                         |The Big Lebowski                            |1998                                        |comedy                                      |
```
Comment (colinhicks) [inline _includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log]: We just need to change the expected output in `tutorial-steps/dev/expected-transient.log` file to match this. This `output-0.log` is spurious

File: _includes/tutorials/transforming/ksql/markup/dev/print-output-topic.adoc [hunk 3]
```diff
@@ -7,5 +7,5 @@ To check that it's working, print out the contents of the output stream's underl
 This should yield the following output:
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/print-output-topic/output-0.log %}</code></pre>
```
Comment (colinhicks) [inline _includes/tutorials/transforming/ksql/markup/dev/print-output-topic.adoc]: Nice catch. Ditto in `transient-query.adoc`

**Pull request merged**